src/app/item-popup/ItemDetails.tsx

```diff
@@ -42,6 +42,10 @@ function mapStateToProps(state: RootState): StoreProps {
 function ItemDetails({ item, extraInfo = {}, defs }: Props) {
   return (
     <div className="item-details-body">
+      {item.type === 'Shaders' && (
```

Comment:
Is this check is reliable enough?

For shaders, `item.type` always seems to be `"Shaders"` and `item.typeName` always seems to be `"Shader"`.

Just unsure of which property to check in which context.

Reply:
item.itemType === 19

https://data.destinysets.com/i/InventoryItem:274843196

Reply:
Thanks @delphiactual 

Have updated the check to use `item.itemCategoryHashes`.

Also took the opportunity to update another check in the same file. Hope that's ok.